config/doctrine/Assignment.orm.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping https://www.doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <entity repository-class="App\Repository\AssignmentRepository" name="App\Entity\Assignment" table="assignments">
+        <id name="id" type="integer" column="id">
+            <generator strategy="AUTO"/>
+            <sequence-generator sequence-name="assignments_id_seq" allocation-size="1" initial-value="1"/>
+        </id>
+        <field name="state" type="string" column="state" length="255" precision="0" scale="0" nullable="false"/>
+        <field name="updatedAt" type="datetime" column="updated_at" nullable="true"/>
+        <many-to-one field="user" target-entity="App\Entity\User" inversed-by="assignments" fetch="LAZY">
```

Comment:
Just a side question:
since we still have one user per assignment should the strategy really be left as `LAZY`?

Reply:
It's not true that we have only one. For example if you use the lambda assignment manager, you can have any number of assignments per user.